src/internal/packager/sbom/catalog.go

```diff
@@ -106,7 +106,7 @@ func Catalog(componentSBOMs map[string]*types.ComponentSBOM, tagToImage map[name
 		currImage++
 	}
 
-	if len(componentSBOMs) > 0 && len(tagToImage) > 0 {
+	if len(componentSBOMs) > 0 || len(tagToImage) > 0 {
```

Comment:
in git history this change might seem odd, can we leave a quick code comment above to explain this conditional?

Reply:
done